e2e/utils.go

```diff
@@ -1378,6 +1383,149 @@ func waitForJobCompletion(c kubernetes.Interface, ns, job string, timeout int) e
 	})
 }
 
+func waitForDeploymentUpdateScale(
```

Comment:
as this code is related to deployment , move all deployment related  code to deployment.go